inference-engine/tests/functional/inference_engine/ngraph_reader/ctc_greedy_decoder_tests.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadCTCGreedyDecoderNetwork) {
+    std::string model = R"V0G0N(
+<net name="ctcgreedydecoder" version="10">
+	<layers>
+		<layer id="0" name="CTCGreedyDecoder/placeholder_port_0" type="Parameter" version="opset1">
+			<data shape="20,8,128" element_type="f32"/>
```

Comment:
I think we shall add another test case when attribute merge_repeated=false is defined.